charts/super-agent/values.yaml

```diff
@@ -91,6 +92,7 @@ helm:
     # installing the chart in case it hits an error.
     # @default -- See `values.yaml`
     upgrade:
```

Comment:
do we need here as well:
 - disableWaitForJobs
and 
```
          rollback:
            disableWait: true
            disableWaitForJobs: true
```
?

Reply:
i miss that!